File: MasterErp.Angular/src/app/components/Purchases/models/PurchaseReturns.ts
import { GeneralOrderDetailsModel } from "../../Inventory/models/GeneralOrderModel ";
import { CreatorModel } from "../../Shared/models/CreatorModel";

export interface PurchaseReturnsModel extends CreatorModel {
    purchaseReturnsId: number | null;
    serialNumber: string;
    docNumber: string;
    purchaseInvoiceId: number | null;
    supplierReturnsVoucherId: number | null;
    totalValue: number | null;
    returnsDate: string | null;
    supplierId: number | null;
    supplierNameEN: string;
    supplierNameAR: string;
    purchaseInvoiceSerialNumber: string;
    notes: string;
    orderDetails: GeneralOrderDetailsModel[];

    nextId?: number | null;
    previousId?: number | null;

    totalCount: number | null;
}

File: MasterErp.Entities/Models/PurchaseReturns.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MasterErp.Entities.Models
{

    public class PurchaseReturns : CreatorModel
    {
        public int PurchaseReturnsId { get; set; }
        public int? SupplierReturnsVoucherId { get; set; }
        public int PurchaseInvoiceId { get; set; }
        public double? TotalValue { get; set; }
        public DateTime? ReturnsDate { get; set; }
        public string Notes { get; set; }



    }
}
